dbms/build.gradle

```diff
@@ -27,6 +27,9 @@ repositories {
         // DBIS Nexus
         url "https://dbis-nexus.dmi.unibas.ch/repository/maven2/"
     }
+    maven {
```

Comment:
Is this required here? There are no other changes to the build file which would use this repository afaict.

Reply:
Moved to #166